host_vars/labs.usegalaxy.org.au.yml

```diff
@@ -0,0 +1 @@
+labs_engine_branch: main
```

Comment:
also need to define some ssh-specific variables such as `extra_keys` and `ssh_config_*`
see `host_vars/galaxy-queue.usegalaxy.org.au.yml` for example